### DISCUSSION TITLE
Add Csound Syntax Highlit to the Language supported

### DISCUSSION THREAD

**Discussion by ghost** [notepadqq/notepadqq#1098] [Ideas]
Hi

It is possible to add Csound to the syntax highlight supported? The Opcodes (or the "words" that need to be highlighted) start from `III Reference` section from the Csound manual:

https://csounds.com/manual/html/index.html

I can't add them, so I ask. 
The Csound community is huge and I think adding support to its opcodes would be a nice thing for Notepadqq

Thanks for now.....

**Comment by rsaxvc** [comment]
It looks like that website is broken. Certificate is bad and link 404s.
